knodle/labeler/CheXpert/environment.yml

```diff
@@ -0,0 +1,58 @@
+name: chexpert-label
+channels:
+  - anaconda
+  - conda-forge
+  - defaults
+dependencies:
+  - blas=1.0=mkl
+  - ca-certificates=2020.1.1=0
+  - certifi=2019.11.28=py36_0
+  - decorator=4.3.0=py_0
+  - docopt=0.6.2=py_1
+  - gcc_linux-64=7.3.0=h553295d_9
```

Comment:
Stuff like this cannot be in Knodle. Good figuring it out though :)

Reply:
shall I just delete the file again then? (:

Reply:
We have to make a decision where to put your code then this can be decided. Definitly helpful for us to have that, thanks :)